src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -74,13 +76,14 @@
     private final ApplicationSettings applicationSettings;
 
     public AuctionRequestFactory(
-            long maxRequestSize,
+            long maxRequestSize, boolean isAccountEnforced,
```

Comment:
Minor. As usual we just repeat config property name, like: `enforceValidAccount`.